insights/parsers/jboss_version.py

```diff
@@ -1,75 +1,127 @@
 """
-JBoss version - File ``$JBOSS_HOME/version.txt``
-================================================
-
-This module provides plugins access to file ``$JBOSS_HOME/version.txt``
-
-Typical content of file ``$JBOSS_HOME/version.txt`` is::
-
-    Red Hat JBoss Enterprise Application Platform - Version 6.4.3.GA
-
-This module parses the file content and stores data in the dict ``self.parsed``.
-The version info can also be got via ``obj.major`` and ``obj.minor``, etc.
-
-Examples:
-    >>> jboss_version.file_path
-    '/home/test/jboss/jboss-eap-6.4/version.txt'
-    >>> jboss_version.raw
-    'Red Hat JBoss Enterprise Application Platform - Version 6.4.3.GA'
-    >>> jboss_version.major
-    6
-    >>> jboss_version.minor
-    4
-    >>> jboss_version.release
-    3
-    >>> jboss_version.version
-    '6.4.3'
-    >>> jboss_version.code_name
-    'GA'
+JBoss version
+=============
+Provide information about the versions of all running Jboss on a system.
 """
-from .. import Parser, parser
-from ..specs import Specs
+
+from collections import namedtuple
+import json
```

Comment:
This is a nitpick, but could you flip these and add a space between them like below? Everything else looks good to me.

```
import json

from collections import namedtuple
```

Reply:
@ryan-blakley - lizhong takes PTO, I update these imports according to your comments. And will merge it soon.